src/audio/smart_amp/dsm_api/inc/dsm_api_public.h

```diff
@@ -14,7 +14,6 @@
 #define DSM_SET_CMD_ID(cmd_id)		DSM_SET_STEREO_CMD_ID(cmd_id)
 
 #define DSM_DEFAULT_NUM_CHANNEL		2
```

Comment:
@ryans-lee PR looks good at first glance but it does too much in one single commit and is hard to understand and review.
Can you please create one commit per single unit of change. 
I would say one commit per items in your commit message.

Reply:
Thanks. I break the first commit into five commits.